src/components/SearchAndFilter/SearchAndFilter.scss

```diff
@@ -111,4 +97,32 @@
     top: 3px;
     z-index: 999;
   }
+
+  &__contextual-menu {
+    display: block;
+
+    .p-contextual-menu__dropdown {
+      padding: 1rem;
+    }
+
+    &-enter,
```

Comment:
These class names seem to be controlled by `CSSTransition` component. They are not consistent with BEM and Vanilla, so it would be worth noting that in a comment here, because quite likely if/when migrating this to Vanilla CSS we would rename them (and break the CSSTransition functionality).

I don't know what to do with such styles in a long run. Technicaly we want to move everything to Vanilla CSS, but also we don't want to put class names dependent on arbitrary react transition library into Vanilla CSS.
Is there any way to configure these class names so they can be renamed in Vanilla "BEM" consistent way?

I'm fine with leaving them as they are for now here, but it should be commented in the code where do they come from and that their names can't be changed because external library controls them.

Reply:
We can supply all the classnames in BEM format: http://reactcommunity.org/react-transition-group/css-transition#CSSTransition-prop-classNames.

I'll update this to use some BEM classes and we can adjust them as needed when we move it to Vanilla.

Reply:
Great, thanks.